ludwig/data/preprocessing.py

```diff
@@ -16,11 +16,13 @@
 # ==============================================================================
 import logging
 from abc import ABC, abstractmethod
+from typing import Any, Dict, List
 
+import dask.dataframe as dd
```

Comment:
See comment below, but either we will ned to import guard this, or (better) create a new type for a Series that can be either Pandas or Dask (iff Dask is installed).

Reply:
Done!